src/Illuminate/Support/helpers.php

```diff
@@ -893,13 +893,13 @@ function tap($value, $callback = null)
      *
      * @param  bool  $boolean
      * @param  \Throwable|string  $exception
-     * @param  string  $message
+     * @param  array  ...$arguments
      * @return void
      */
-    function throw_if($boolean, $exception, $message = '')
+    function throw_if($boolean, $exception, ...$arguments)
```

Comment:
Uh... I've got that backwards.

I was simply saying that I'd use `$parameters` because that's what we use all over in our `__call` methods. I wasn't thinking from a semantic perspective.

From a semantic perspective, `$arguments` actually makes _more_ sense.

So I take back what I said originally. I now believe that `$arguments` is actually a better name :see_no_evil: